blog/authors.yml

```diff
@@ -2,4 +2,9 @@ jacoobes:
   name: jacoobes
   title: Head Dev
   url: https://github.com/jacoobes
-  image_url: https://github.com/jacoobes.png
\ No newline at end of file
+  image_url: https://github.com/jacoobes.png
+Sr Izan:
+  name: Sr Izan
+  title: Contributor
+  url: https://github.com/SrIzan10
+  image_url: https://github.com/SrIzan10.png
```

Comment:
If you can, please add me there, I might post a text in future. If you don't want to do, dw I'll when I want to post something

Reply:
Yeah cool